packages/mermaid/src/diagrams/railroad/railroadDiagram.ts

```diff
@@ -0,0 +1,15 @@
+import { DiagramDefinition } from '../../diagram-api/types.js';
+// @ts-ignore: jison doesn't export types
+import parser from './railroadParser.jison';
+import { db } from './railroadDB.js';
+import renderer from './railroadRenderer.js';
+// import { prepareTextForParsing } from './railroadUtils.js';
+
+// const originalParse = parser.parse.bind(parser);
+// parser.parse = (text: string) => originalParse(prepareTextForParsing(text));
```

Comment:
Are there necessary?

Reply:
Sounds strange, but yes. I needed such a hack inside sankey, to prepare text before parsing, and I hadn't have a callback for that. Maybe we need to re-implement this, but I've left it here intentionally